.pre-commit-config.yaml

```diff
@@ -0,0 +1,99 @@
+---
+fail_fast: false
+default_stages:
```

Comment:
running `pre-commit migrate-config` will fix the stages.

Reply:
strange

```
21:36:45 coal  feature/RES-118-refactor-to-typescript ⬢ v23.11.0 ☁️  default/ ☸️ kind-agents-devenv  
behave.tracking-client-lib ➜ pre-commit migrate-config                        
Configuration has been migrated.
21:36:57 coal  feature/RES-118-refactor-to-typescript [!] ⬢ v23.11.0 ☁️  default/ ☸️ kind-agents-devenv  
behave.tracking-client-lib ➜ git diff
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index 9877ebb..712ccdc 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -1,8 +1,8 @@
 ---
 fail_fast: false
 default_stages:
-  - commit
-  - push
+  - pre-commit
+  - pre-push
 
 repos:
   - repo: https://github.com/thlorenz/doctoc
```

Reply:
should I just make these changes manually? I checked my pre-commit version and I have `3.8.0` locally, newest is `4.2.0` ... could it be related to that, I guess I should update?